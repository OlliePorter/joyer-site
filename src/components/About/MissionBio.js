import React from 'react';
import Typography from '@material-ui/core/Typography';
import ChatBubbleOutlineIcon from '@material-ui/icons/ChatBubbleOutline';
import { makeStyles } from '@material-ui/core/styles';
import theme from '../../theme';

const useStyles = makeStyles({
    aboutText: {
      color: theme.palette.primary.altDark,
      padding: '1.3em',
      paddingTop: '1.6em',
      fontWeight: 300,
      width: '42%',
      [theme.breakpoints.down('sm')]: {
        width: '100%',
      },
    },
    button: {
      width: '179px',
      color: 'white',
      backgroundColor: theme.palette.primary.main,
      height: '52px',
      fontSize: '17px',
      borderRadius: '50px',
      textTransform: 'none',
      boxShadow: 'none',
      fontWeight: '300',
      "&:hover": {
        background: theme.palette.primary.dark,
        boxShadow: 'none',
      },
    },
    missionBackground: {
      backgroundColor: theme.palette.primary.altLight,
      paddingTop: '5em',
      paddingBottom: '5em',
      display: 'flex',
      flexDirection: 'column',
      justifyContent: 'center',
      alignItems: 'center',
      [theme.breakpoints.down('sm')]: {
        padding: '3em',
        paddingTop: '5em',
      },
    },
    circleBackground: {
      width: '100px',
      height: '100px',
      backgroundColor: theme.palette.primary.main,
      borderRadius: '50%',
      display: 'flex',
      justifyContent: 'center',
      alignItems: 'center',
    },
    speechIcon: {
      color: theme.palette.primary.light,
      transform: 'scale(1.3)',
    }
});

export default function MissionBio() {
  const classes = useStyles();

    return (
    <>
      <div className={classes.missionBackground}>
        <div className={classes.circleBackground}>
          <ChatBubbleOutlineIcon className={classes.speechIcon}/>
        </div>
        <Typography component="p" className={classes.aboutText} align="center">
        We live in unprecedented times. In a matter of days, a global pandemic turned the world upside down and fundamentally altered our daily lives. Naturally, governments around the world enacted a social distancing policy to reduce the spread of the virus, but these new measures have already begun to adversely affect the mental health of the global population. Enter JOYER, a wellbeing app for building high quality social connections. It’s the best way to maintain strong relationships and bring balance to your headspace. We developed JOYER to become the greatest tool in everyone’s social arsenal — a personal CRM which brings us all closer to those who matter.
        </Typography>
      </div>
    </>
    );
}