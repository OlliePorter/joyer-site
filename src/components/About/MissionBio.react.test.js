import React from 'react';
import MissionBio from './MissionBio';
import renderer from 'react-test-renderer';

it('Cover renders correctly', () => {
    const tree = renderer.create(
      <MissionBio />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });