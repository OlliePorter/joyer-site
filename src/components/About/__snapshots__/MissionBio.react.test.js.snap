// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cover renders correctly 1`] = `
<div
  className="makeStyles-missionBackground-3"
>
  <div
    className="makeStyles-circleBackground-4"
  >
    <svg
      aria-hidden="true"
      className="MuiSvgIcon-root makeStyles-speechIcon-5"
      focusable="false"
      role="presentation"
      viewBox="0 0 24 24"
    >
      <path
        d="M20 2H4c-1.1 0-2 .9-2 2v18l4-4h14c1.1 0 2-.9 2-2V4c0-1.1-.9-2-2-2zm0 14H6l-2 2V4h16v12z"
      />
    </svg>
  </div>
  <p
    className="MuiTypography-root makeStyles-aboutText-1 MuiTypography-body1"
  >
    We live in a hyperconnected world driven by endless chimes and beeps of continuously arriving text messages, Tweets and Facebook updates. This interconnectivity has revolutionised the way we communicate, but today many of us find it difficult to maintain close relationships with everyone in our social circle. This is because of the busyness of our daily lives, extensive social media connections and no standardised tool to promote face-to-face contact.
  </p>
  <p
    className="MuiTypography-root makeStyles-aboutText-1 MuiTypography-body1"
  >
    We believe in a world where social still has a soul. Where the quality of our interpersonal relationships are valued higher than extensive social circles.
  </p>
  <p
    className="MuiTypography-root makeStyles-aboutText-1 MuiTypography-body1"
  >
    JOYER is a social app which promotes the authenticity of connection through memorable offline interactions, relationship cultivation and contextual reminders. We hope you love it as much as we do.
  </p>
  <button
    className="MuiButtonBase-root MuiButton-root MuiButton-contained makeStyles-button-2"
    disabled={false}
    onBlur={[Function]}
    onDragLeave={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex={0}
    type="button"
  >
    <span
      className="MuiButton-label"
    >
      Find out more
    </span>
  </button>
</div>
`;
